freqtrade/freqtradebot.py

```diff
@@ -1272,7 +1272,8 @@ def handle_cancel_enter(self, trade: Trade, order: Dict, reason: str) -> bool:
             # to the order dict acquired before cancelling.
             # we need to fall back to the values from order if corder does not contain these keys.
             trade.amount = filled_amount
-            # TODO-lev: Check edge cases, we don't want to make leverage > 1.0 if we don't have to
+            # * Check edge cases, we don't want to make leverage > 1.0 if we don't have to
+            # * (for leverage modes which aren't isolated futures)
```

Comment:
In isolated futures, the leverage is always the amount of collateral you put in, if you add more collateral, it goes in with the same amount of leverage, so your leverage doesn't change depending on if an order is filled all the way or not